src/Algebra/Local/Structures.agda

```diff
@@ -0,0 +1,60 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Local algebraic structures
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Local.Structures where
+
+open import Level using (Level; _⊔_) renaming (suc to lsuc)
+open import Data.Product using (∃-syntax; _×_)
+open import Algebra.Core using (Op₁; Op₂)
+open import Algebra.Definitions using (Invertible)
+open import Algebra.Structures using (IsCommutativeRing)
+open import Relation.Binary.Core using (Rel)
+open import Relation.Binary.Structures using (IsEquivalence; IsApartness)
+open import Relation.Binary.Definitions using (Tight)
+import Relation.Binary.Properties.Apartness as A
+open import Relation.Nullary using (¬_)
+
+
+module _
+  {c ℓ₁ ℓ₂} {Carrier : Set c}
+  (_≈_ : Rel Carrier ℓ₁)
+  (_#_ : Rel Carrier ℓ₂)
```

Comment:
We can play the same trick as in `Algebra.Structures` and lift `{a ℓ₁ ℓ₂} {A : Set a}  (_≈_ : Rel Carrier ℓ₁)  (_#_ : Rel Carrier ℓ₂)` up to be parameters of the top-level module, which allows users to import all it's contents already parameterised :+1:  The other arguments should then just be part of the main definition.

Reply:
Yep done.